v2/concordium/types.proto

```diff
@@ -503,6 +503,58 @@ message Cooldown {
   CooldownStatus status = 3;
 }
 
+// The unique id of a token. The identifier is the position in the token table.
+message TokenId {
+  uint64 value = 1;
+}
+
+// The unique identifier of a token module.
+message TokenModuleId {
+  uint64 value = 1;
```

Comment:
It would be nice to use a module reference, similar to smart contract modules here, since it would make the identifier independent on the network and the order in which they were deployed.